pylint/lint.py

```diff
@@ -72,6 +72,9 @@
 from pylint.reporters.ureports import nodes as report_nodes
 
 
+FULL_VERSION = '%%(prog)s %s\nastroid %s\nPython %s' % (
+    version, astroid_version, sys.version)
+
```

Comment:
`FULL_VERSION` is available at module level for others to use, but it won't have `%(prog)s` substituted with pylint. Will that be a problem. Also put it here instead of __pkginfo__ with other versions because it isn't pkginfo related.